### PR TITLE
Improving README.md with compose example & where to find snapshot failure diffs

### PR DESCRIPTION
**Pull request by rojanthomas**
Hi, I think the README can be improved to mention the ability to snapshot test composables & where to find snapshot failure diff images.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Paparazzi
 ========
 
-An Android library to render your application screens without a physical device or emulator.
+An Android library to snapshot test your views or composables without a physical device or emulator.
```
Comment (jrodbx) [inline README.md:4]: revert

File: README.md [hunk 2]
```diff
@@ -13,14 +13,21 @@ class LaunchViewTest {
   )
 
   @Test
-  fun simple() {
+  fun simpleViewTest() {
```
Comment (jrodbx) [inline README.md:16]:
```suggestion
  fun launchView() {
```

File: README.md [hunk 3]
```diff
@@ -13,14 +13,21 @@ class LaunchViewTest {
...
     val view = paparazzi.inflate<LaunchView>(R.layout.launch)
     // or...
     // val view = LaunchView(paparazzi.context)
 
     view.setModel(LaunchModel(title = "paparazzi"))
     paparazzi.snapshot(view)
   }
+
+  @Test
+  fun simpleComposableTest() {
```
Comment (jrodbx) [inline README.md:26]:
```suggestion
  fun launchComposable() {
```

File: README.md [hunk 4]
```diff
@@ -46,8 +53,7 @@ Saves snapshots as golden values to a predefined source-controlled location
 $ ./gradlew sample:verifyPaparazziDebug
 ```
 
-Runs tests and verifies against previously-recorded golden values.
-
+Runs tests and verifies against previously-recorded golden values. Snapshot test failure diffs can be found at `sample/out/failures`.
```
Comment (jrodbx) [inline README.md:56]:
```suggestion
Runs tests and verifies against previously-recorded golden values.  Failures generate diffs at `sample/out/failures`.
```

**Review by jrodbx — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (rojanthomas) [reply README.md]: Reverted

**Comment by rojanthomas** [comment]
@jrodbx Thanks for the review, I've made the changes you've suggested. 😄

**Pull request merged**